src/Features/Core/Portable/CodeRefactorings/FixAllOccurences/FixAllCodeRefactoringCodeAction.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings
+{
+    internal sealed class FixAllCodeRefactoringCodeAction : CodeAction
+    {
+        internal readonly FixAllState FixAllState;
+        private bool _showPreviewChangesDialog;
```

Comment:
mutable?  seems... suspect.

Reply:
Matched the existing implementation: https://github.com/dotnet/roslyn/blob/e1629052738683dac234a4d75bb1d1c442c46d7d/src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs#L23

This is mutable primarily for test path: https://github.com/dotnet/roslyn/blob/e1629052738683dac234a4d75bb1d1c442c46d7d/src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs#L96-L108